plugins/chat/spec/fabricators/chat_fabricator.rb

```diff
@@ -63,7 +63,7 @@
 Fabricator(:chat_message_without_service, class_name: "Chat::Message") do
   user
   chat_channel
-  message { Faker::Lorem.paragraph_by_chars(number: 500).gsub("...", "…") }
+  message { Faker::Lorem.paragraph_by_chars(number: 500).gsub("...", "…").gsub("..", "…") }
```

Comment:
Shorter, but not sure if clearer 🤔 

```suggestion
  message { Faker::Lorem.paragraph_by_chars(number: 500).gsub(/[.]{2,3}/, "…") }
```

Reply:
yes :/ I think I prefer the other one but thanks